fastlane/Fastfile

```diff
@@ -236,6 +237,67 @@ platform :android do
       skip_upload_screenshots: true
     )
   end
+
+  desc "Deploy HybridApp to Google Play production at 1% rollout"
+    lane :upload_google_play_production_hybrid_rollout do
+      # Google is very unreliable, so we retry a few times
+      ENV["SUPPLY_UPLOAD_MAX_RETRIES"]="5"
+      google_play_track_version_codes(
+        package_name: "org.me.mobiexpensifyg",
+        json_key: './android/app/android-fastlane-json-key.json',
+        track: 'internal'
+      )
+      upload_to_play_store(
+        package_name: "org.me.mobiexpensifyg",
+        json_key: './android/app/android-fastlane-json-key.json',
+        version_code: ENV["VERSION"].to_i,
+        track: 'internal',
+        track_promote_to: 'production',
+        rollout: '0.01',
```

Comment:
So when submitting for review, it will auto-rollout to 1% after approval?

Reply:
Yes, correct!